contracts/rewards/AutoCompoundingRewards.sol

```diff
@@ -89,6 +94,11 @@ contract AutoCompoundingRewards is IAutoCompoundingRewards, ReentrancyGuardUpgra
      */
     event ProgramEnabled(Token indexed pool, bool status, uint256 remainingRewards);
 
+    /**
+     * @dev triggered when the auto trigger count is updated
```

Comment:
Nitpick: I'd change the comment to `auto-trigger`, like you did in other places

Reply:
done